FordLabs/retroquest
# Change the character count to yellow between 50 and 1 characters remaining

## Overview
This PR will change the characters remaining counter to yellow when between 50 and 1 characters remaining. At 0 characters it will turn red. 

Previously the character count turned red below 50 characters.

### Demo
<img width="400" alt="Screen Shot 2021-05-05 at 10 33 05 AM" src="https://user-images.githubusercontent.com/5607510/117158878-cdd1ba00-ad8d-11eb-9212-59204ccd8dfd.png">
<img width="400" alt="Screen Shot 2021-05-05 at 10 32 57 AM" src="https://user-images.githubusercontent.com/5607510/117158880-cdd1ba00-ad8d-11eb-8bbf-3f234bd6a669.png">

<img width="400" alt="Screen Shot 2021-05-05 at 10 36 21 AM" src="https://user-images.githubusercontent.com/5607510/117158861-cb6f6000-ad8d-11eb-89c5-23e673b2c415.png">
<img width="400" alt="Screen Shot 2021-05-05 at 10 35 32 AM" src="https://user-images.githubusercontent.com/5607510/117158874-cca08d00-ad8d-11eb-9cb2-09a6392469a9.png">


### Notes
Fixes #287

## Testing Instructions
- Start the frontend and backend
- Login or create a board
- Enter 204 characters into a thought, the character count should be the default color
- Enter 206 characters into a thought, the character count should be yellow
- Enter 255 characters into a thought, the character count should be red
